python/llm/scripts/README.md

```diff
@@ -1,11 +1,41 @@
-# README
-Script will first check Python installation, then site-packages (transformers, torch and bigdl-llm), and hardware or operating system and finally check xpu-related issue. 
-## Linux
-### How to use
-Run `bash env-check.sh` on linux. 
-### Sample output
-Here is the output of the scipt on arc12 with GPU installed properly.
-Note that when only CPU is aviailable, we can ignore xpu related checks. 
+#  Utility Scripts
+
+
+## Env-Check
+
+The **Env-Check** scripts  ([env-check.sh](./env-chec.sh), [env-check.bat](./env-chec.bat)) are designed to verify your `bigdl-llm` installation and runtime environment. These scripts can help you ensure your environment is correctly set up for optimal performance. You can include the script's output when reporting issues on [BigDL Github Issues](https://github.com/intel-analytics/BigDL/issues) for easier troubleshooting.
+
+> Note: These scripts verify python installation, check for necessary packages and environmental variables, assess hardware or operating system compatibility, and identify any XPU-related issues. 
+
+### Install extra dependency
+
+* On Linux, the script uses a tool named `xpu-smi`. It is a convinent tool the monitor ths status of the GPUs. If you're runing LLMs on GPUs on Linux, we recommend installing `xpu-smi`. Run below command to install:
+```
+sudo apt install xpu-smi
+```
+* On Windows, you can ignore the sections in `xpu-smi.exe` if you didn't install it. You can always use **Windows Task Manager** to monitor the status of GPUs on Windows.   
```

Comment:
Can user install `xpu-smi.exe` on WIndows? If yes, how?

Reply:
Yes. See https://intel.github.io/xpumanager/WIN_README.html But we didn't have time to try it yet... we plan to add this section later. Now on Windows we mostly use graphical monitoring like Task manager.